app/src/main/java/protect/card_locker/GroupShortcutConfigure.java

```diff
@@ -0,0 +1,96 @@
+package protect.card_locker;
+
+import android.database.Cursor;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.content.pm.ShortcutInfoCompat;
+import androidx.core.content.pm.ShortcutManagerCompat;
+import androidx.recyclerview.widget.DefaultItemAnimator;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+/**
+ * The configuration screen for creating a shortcut.
+ */
+public class GroupShortcutConfigure extends AppCompatActivity implements GroupSelectCursorAdapter.GroupAdapterListener {
+    static final String TAG = "Catima";
+    final DBHelper mDb = new DBHelper(this);
+
+    @Override
+    public void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+
+        // Set the result to CANCELED.  This will cause nothing to happen if the
+        // aback button is pressed.
+        setResult(RESULT_CANCELED);
+
+        setContentView(R.layout.main_activity);
```

Comment:
I created a new clean base layout for this, please use `R.layout.simple_toolbar_list_activity` instead (see the changes to `CardShortcutConfigure.java`). Then you also don't have to hide the FAB and mess around with the LayoutManager and all